README.md

```diff
@@ -321,17 +318,18 @@ Other examples are provided in the [examples](examples) directory.
 You need first to *clone* this repository and run:
 
 ```bash
-cd zombienet/javascript
-npm install
-npm run build
+❯ cd zombienet/javascript
+❯ npm install
+❯ npm run build
 ```
 
 ### Download and install needed artifacts (optional)
 
 For an easier and faster setup of your local environment, run:
 
 ```bash
-node dist/cli.js setup <binaries>
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js <binaries>
```

Comment:
```suggestion
❯ node ./cli/dist/cli.js setup <binaries>
```

Reply:
For consistency reason I suggest:
```ts
❯ cd zombinet/javascript
❯ npm run zombie -- setup <binaries>
```

Reply:
@wirednkod, will stick with your suggestion then around these docs.